modules/fts/pages/fts-scoring-score-none.adoc

```diff
@@ -0,0 +1,20 @@
+= Score:none
+
+Scoring adds performance and reduces overhead. Sometimes when you have range predicates in the query, you can avoid scoring.
```

Comment:
"adds performance and reduces overhead" what does this mean?

Reply:
Update: needs to be updated to talk about scoring and compute.